### PR TITLE
Locale for Mention Me Tags

### PR DESCRIPTION
**Pull request by edhgoose**
The locale parameter is missing on the Mention Me tags - this will add it to the Referee and Referrer tags (it is currently present on Dashboard).

### CODE REVIEW

**Comment by edhgoose** [comment]
@peterfronc and @emreb, is it possible for one of you to review and merge this? We have a client who'd really benefit from it.

**Comment by peterfronc** [comment]
@edhgoose I will test it and release latest tomorrow morning.

File: libraries/mentionme/refereetag/v1/Tag.js [hunk 1]
```diff
@@ -66,7 +71,9 @@ qubit.opentag.LibraryTag.define("mentionme.refereetag.v1.Tag", {
 			paramObj["fullname"] = "" + this.valueForToken("fullname");
 		if (("" + this.valueForToken("implementation")).length)
 			paramObj["implementation"] = "" + this.valueForToken("implementation");
-
+		if (("" + this.valueForToken("locale")).length)
```
Comment (peterfronc) [inline libraries/mentionme/refereetag/v1/Tag.js]: This actually checks the string length of result but 'valueForToken' returns any objects (including undefined if unset!). Just a reminder to make sure it's intended behavior.

File: libraries/mentionme/refereetag/v1/Tag.js [hunk 2]
```diff
@@ -66,7 +71,9 @@ qubit.opentag.LibraryTag.define("mentionme.refereetag.v1.Tag", {
...
+			paramObj["locale"] = "" + this.valueForToken("locale");
```
Comment (peterfronc) [inline libraries/mentionme/refereetag/v1/Tag.js]: No need really to convert to string but won't hurt neither.

**Comment by peterfronc** [comment]
@edhgoose It's now live and available to use.
screen shot 2018-07-06 at 10 32 35

**Pull request merged**